benchmarks/setup/newBenchmark.sh

```diff
@@ -1,4 +1,27 @@
 #!/bin/bash
+
+GO_OS=${GO_OS:-"linux"}
+
+function detect_os {
+    # Detect the OS name
+    case "$(uname -s)" in
+      Darwin)
+        host_os=darwin
+        ;;
+      Linux)
+        host_os=linux
+        ;;
+      *)
+        echo "Unsupported host OS. Must be Linux or Mac OS X." >&2
```

Comment:
I'm not sure about this. We have co-worker(s) working on Windows as well AFAIK. 

@pihme does the current sed implementation of this newBenchmark script work on your machine?

Reply:
It does work. I have to use WSL (Windows Subsystem for Linux) anyway, and it registers properly as Linux i that mode.

Reply:
@pihme ok, so the change here would also still work on WSL?

Reply:
Yes, I tested it.

Reply:
Thanks